### PR TITLE
switching out to poise-python

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

**Comment by rshade** [comment]
@djoos since python is deprecated and chef13 just dropped in the current channel, I went ahead and switched these out.

File: resources/agent_python.rb [hunk 1]
```diff
@@ -34,3 +34,4 @@
 attribute :browser_monitoring_auto_instrument, :kind_of => [TrueClass, FalseClass], :default => true
 attribute :cross_application_tracer_enable, :kind_of => [TrueClass, FalseClass], :default => true
 attribute :feature_flag, :kind_of => String, :default => nil
+attribute :python, :kind_of => String, :default => nil
```
Comment (djoos) [inline resources/agent_python.rb]: Is this attribute actually used? Thanks in advance!

Comment (rshade) [inline resources/agent_python.rb]: sorry forgot to put it in the provider. Good catch. Thanks man.

Comment (djoos) [inline resources/agent_python.rb]: Great, thank you - aiming to get this merged in ASAP!

Comment (rshade) [inline resources/agent_python.rb]: do you have a .kitchen.yml you are testing against?

Comment (rshade) [inline resources/agent_python.rb]: never mind, I re-read testing :)

Comment (djoos) [inline resources/agent_python.rb]: Yes, here's the cookbook's [.kitchen.yml](https://github.com/djoos-cookbooks/newrelic/blob/master/.kitchen.yml)...

Comment (rshade) [inline resources/agent_python.rb]: I will drop the kitchen converge stuff in here, so it shows

**Comment by JoshuaEstes** [comment]
Any idea on the timeframe for this? I would like to continue to use this cookbook, but having this require python instead of poise-python is starting to cause real problems for me.

**Comment by rshade** [comment]
@JoshuaEstes I am trying to get the tests to work. If you guys would like to pull my fork and give me any ideas on the issues I am running into with the python recipe, that would be awesome.

**Comment by djoos** [comment]
@rshade: I also spotted issues when running the integration tests - I had an initial look (to be honest: there were some failures not related to the changes in this PR as well!), but had to park it for now.

@JoshuaEstes once we can get all tests to pass, we can get this PR merged and shipped!

**Comment by rshade** [comment]
Finished in 1.87 seconds (files took 1.73 seconds to load)
       20 examples, 0 failures

       Finished verifying <python-agent-recipe-ubuntu-1204> (0m17.42s).

**Comment by rshade** [comment]
`Kitchen is finished. (30m54.04s)` @djoos  woot

**Comment by djoos** [comment]
Thanks for your contribution @rshade!

**Comment by rshade** [comment]
any day @djoos. Thanks for a great cookbook

**Comment by jyaworski** [comment]
Can we get a release soon so we can use this with Chef 13?

**Comment by djoos** [comment]
@jyaworski v2.27.0 just got released to the Chef Supermarket!

**Comment by JoshuaEstes** [comment]
Thanks for getting this out guys!

**Pull request merged**